lib/router.js

```diff
@@ -182,6 +185,7 @@ router._getParams = function(req) {
 };
 
 router.sendResponse = function(res, payload, httpCode) {
+  metrics.processResponse(res._request, httpCode, payload, (new Date()) - res._startDate);
```

Comment:
Worth using [`process.hrtime`](https://nodejs.org/api/process.html#process_process_hrtime_time) for improved accuracy?